shell/browser/native_window_views.cc

```diff
@@ -649,8 +652,9 @@ void NativeWindowViews::Maximize() {
   if (IsTranslucent()) {
     // If a window is translucent but not transparent on Windows,
```

Comment:
nit: This comment needs to be updated also to reflect the change made in the conditional

Reply:
done.